src/poker.js

```diff
@@ -1,10 +1,72 @@
-function winningPair() {}
+function winningPair(pair1, pair2) {
+  if (pair1[0] === pair1[1]) {
+    if (pair2[0] === pair2[1]) {
+      const fig = [] // an array that is used to turn figures into numbers
```

Comment:
"A comment is an apology"

If you feel like you have to explain what something is, it usually means it can be named better. Comments should describe why something exists rather than what it is.

In this case you could rename this variable to `cardScore` or something